src/Disqord.Core/Enums/GuildScheduledEventEntityType.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Disqord
+{
+    public enum GuildScheduledEventEntityType : byte
+    {
+        None,
+
+        StageInstance,
```

Comment:
To be consistent with the lib's naming.
```suggestion
        Stage,
```